db-scripts/10_initial_table_load.sql

```diff
@@ -222,3 +1228,98 @@ VALUES (1,'Route 1')
 ,(23,'Route 23')
 ,(24,'Route 24')
 ,(25,'Route 25');
+
+
+/*************************************\
+** REGION_ROUTE_XREF
+\*************************************/
+INSERT INTO region_route_xref (region_id, route_id, game_id)
+VALUES (1,1,1)
+,(1,2,1)
+,(1,3,1)
+,(1,4,1)
+,(1,5,1)
+,(1,6,1)
+,(1,7,1)
+,(1,8,1)
+,(1,9,1)
+,(1,10,1)
+,(1,11,1)
+,(1,12,1)
+,(1,13,1)
+,(1,14,1)
+,(1,15,1)
+,(1,16,1)
+,(1,17,1)
+,(1,18,1)
+,(1,19,1)
+,(1,20,1)
+,(1,21,1)
+,(1,22,1)
+,(1,23,1)
+,(1,24,1)
+,(1,25,1)
+,(1,1,2)
+,(1,2,2)
+,(1,3,2)
+,(1,4,2)
+,(1,5,2)
+,(1,6,2)
+,(1,7,2)
+,(1,8,2)
+,(1,9,2)
+,(1,10,2)
+,(1,11,2)
+,(1,12,2)
+,(1,13,2)
+,(1,14,2)
+,(1,15,2)
+,(1,16,2)
+,(1,17,2)
+,(1,18,2)
+,(1,19,2)
+,(1,20,2)
+,(1,21,2)
+,(1,22,2)
+,(1,23,2)
+,(1,24,2)
+,(1,25,2)
+,(1,1,3)
+,(1,2,3)
+,(1,3,3)
+,(1,4,3)
+,(1,5,3)
+,(1,6,3)
+,(1,7,3)
+,(1,8,3)
+,(1,9,3)
+,(1,10,3)
+,(1,11,3)
+,(1,12,3)
+,(1,13,3)
+,(1,14,3)
+,(1,15,3)
+,(1,16,3)
+,(1,17,3)
+,(1,18,3)
+,(1,19,3)
+,(1,20,3)
+,(1,21,3)
+,(1,22,3)
+,(1,23,3)
+,(1,24,3)
+,(1,25,3);
+
+
+
+/*************************************\
+** POKEMON_SPAWN_RATE (DRAFT)
+\*************************************/
+INSERT INTO pokemon_spawn_rate (pokemon_id, region_route_xref_id, location, min_level, max_level, rate)

```

Comment:
@tylerjamesf - this attempts to rebuild the existing spawn rates as we understand them today. If this is something we like we can continue to populate. This doesn't account for Morning/Day/Night themes yet but could easily be expanded to include.